sdk/Cargo.toml

```diff
@@ -25,6 +25,7 @@ file_io = ["openssl"]
 serialize_thumbnails = []
 xmp_write = ["xmp_toolkit"]
 no_interleaved_io = ["file_io"]
+fetch_remote_manifests = ["file_io"]
```

Comment:
Ok, but, as we said, let's move away from adding features

Reply:
Yes, we do not have a way to control this yet.